v5/v5_with_library/v5_stable.ino

```diff
@@ -733,6 +739,8 @@ bool deleteMessage(int index) {
   } else if (index < 0) {
     index *= -1;
   }
+  if (pending_test_message)
+    return true;
```

Comment:
Please explain me this? it will cause the loop, fix it i think you should assign it false not true.

Reply:
This is not in getResponse function it's in delete message.